### PR TITLE
New package: android-studio

### PR DESCRIPTION
**Pull request by jcjordyn130**

### CODE REVIEW

**Comment by Vaelatern** [comment]
Supersedes #5232

**Comment by jcjordyn130** [comment]
Wow this is taking a while to get merged... :P

**Comment by Vaelatern** [comment]
It might take less time if A) You indented that function, B) You didn't need to include a .desktop file, C) Didn't have a travis failure you could have seen any time in the last 10 days and done something about.

**Comment by jcjordyn130** [comment]
I saw the travis failure, it's xlint saying that revision must not be 0 but its 1.

**Comment by Vaelatern** [comment]
A) and B) and a D) Name the commit correctly? See the end of the Manual.

**Comment by jcjordyn130** [comment]
@Vaelatern Done, and I also removed the desktop file.

File: srcpkgs/android-studio/template [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+depends="alsa-lib freetype libXrender libXtst gtk+ libGL"
+short_desc="The official Android IDE"
+maintainer="Jordyn Carattini <onlinecloud1@gmail.com>"
+only_for_archs="x86_64 i686"
+distfiles="https://dl.google.com/dl/android/studio/ide-zips/${version}.${_studio_rev}/android-studio-ide-${_studio_build}-linux.zip"
+checksum="95014a19786c076b31f0ea980eb42a163e9738356ec08a0878b7d67b6b7ec76e"
+license="Apache-2.0"
+homepage="http://tools.android.com/"
+nopie=yes
+nostrip=yes
```
Comment (Vaelatern) [inline srcpkgs/android-studio/template]: Reason for nostrip?

File: srcpkgs/android-studio/template [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+pkgname=android-studio
+version=2.2.2
+revision=1
+# _studio_build and _studio_rev are for downloading the zip from dl.google.com
+_studio_build=145.3360264
+_studio_rev=0
+create_wrksrc=yes
+build_style=fetch
+hostmakedepends="unzip"
+depends="alsa-lib freetype libXrender libXtst gtk+ libGL"
```
Comment (Vaelatern) [inline srcpkgs/android-studio/template]:
```
=> android-studio-2.2.2_1: running pre-pkg hook: 03-rewrite-python-shebang ...
=> android-studio-2.2.2_1: running pre-pkg hook: 04-generate-runtime-deps ...
   SONAME: libc.so.6 <-> glibc>=2.8_1
   SONAME: libjli.so <-> openjdk-jre>=8u20_1
   SONAME: libawt_xawt.so <-> openjdk-jre>=8u20_1
   SONAME: libdl.so.2 <-> glibc>=2.8_1
   SONAME: libm.so.6 <-> glibc>=2.8_1
   SONAME: libpthread.so.0 <-> glibc>=2.8_1
   SONAME: ld-linux-x86-64.so.2 <-> glibc>=2.8_1
   SONAME: libasound.so.2 <-> alsa-lib>=1.0.20_1
   SONAME: libawt.so <-> openjdk-jre>=8u20_1
   SONAME: libjava.so <-> openjdk-jre>=8u20_1
   SONAME: libjvm.so <-> openjdk-jre>=8u20_1
   SONAME: libfreetype.so.6 <-> freetype>=2.3.9_1
   SONAME: libgcc_s.so.1 <-> libgcc>=4.4.0_1
   SONAME: libnio.so <-> android-studio (ignored)
   SONAME: libnet.so <-> android-studio (ignored)
   SONAME: libverify.so <-> android-studio (ignored)
   SONAME: libX11.so.6 <-> libX11>=1.2_1
   SONAME: libXext.so.6 <-> libXext>=1.0.5_1
   SONAME: libthread_db.so.1 <-> glibc>=2.8_1
   SONAME: libXrender.so.1 <-> libXrender>=0.9.4_1
   SONAME: libXtst.so.6 <-> libXtst>=1.0.3_1
   SONAME: libXi.so.6 <-> libXi>=1.2.1_1=> android-studio-2.2.2_1: running pre-pkg hook: 03-rewrite-python-shebang ...
=> android-studio-2.2.2_1: running pre-pkg hook: 04-generate-runtime-deps ...
   SONAME: libc.so.6 <-> glibc>=2.8_1
   SONAME: libjli.so <-> openjdk-jre>=8u20_1
   SONAME: libawt_xawt.so <-> openjdk-jre>=8u20_1
   SONAME: libdl.so.2 <-> glibc>=2.8_1
   SONAME: libm.so.6 <-> glibc>=2.8_1
   SONAME: libpthread.so.0 <-> glibc>=2.8_1
   SONAME: ld-linux-x86-64.so.2 <-> glibc>=2.8_1
   SONAME: libasound.so.2 <-> alsa-lib>=1.0.20_1
   SONAME: libawt.so <-> openjdk-jre>=8u20_1
   SONAME: libjava.so <-> openjdk-jre>=8u20_1
   SONAME: libjvm.so <-> openjdk-jre>=8u20_1
   SONAME: libfreetype.so.6 <-> freetype>=2.3.9_1
   SONAME: libgcc_s.so.1 <-> libgcc>=4.4.0_1
   SONAME: libnio.so <-> android-studio (ignored)
   SONAME: libnet.so <-> android-studio (ignored)
   SONAME: libverify.so <-> android-studio (ignored)
   SONAME: libX11.so.6 <-> libX11>=1.2_1
   SONAME: libXext.so.6 <-> libXext>=1.0.5_1
   SONAME: libthread_db.so.1 <-> glibc>=2.8_1
   SONAME: libXrender.so.1 <-> libXrender>=0.9.4_1
   SONAME: libXtst.so.6 <-> libXtst>=1.0.3_1
   SONAME: libXi.so.6 <-> libXi>=1.2.1_1
```
So you don't need libXtst, or libXrender, or freetype, or alsa-lib.
I wonder if you actually need gtk+ or libGL either

File: srcpkgs/android-studio/template (hunk 1 above)
Comment (jcjordyn130) [inline srcpkgs/android-studio/template]: Post build hook failed in xbps-src.

File: srcpkgs/android-studio/template (hunk 2 above)
Comment (jcjordyn130) [inline srcpkgs/android-studio/template]: The emulator uses libGL.

**Pull request closed without merging**